src/interfaces/chain.cpp

```diff
@@ -35,6 +150,34 @@ class ChainImpl : public Chain
         return std::move(result);
     }
     std::unique_ptr<Chain::Lock> assumeLocked() override { return MakeUnique<LockImpl>(); }
+    bool findBlock(const uint256& hash, CBlock* block, int64_t* time, int64_t* time_max) override
```

Comment:
nit: none of the callers use more than one of the out args, so splitting this up would make the call sites simpler/clearer.